css/style.css

```diff
@@ -27,6 +27,14 @@ h1, h2, h3, h4 {
   border-top: solid 1px #e2e2e2;
 }
 
+.columns {
+  min-height: 1px;
+}
+
+.columns li {
```

Comment:
I do not see this as a good solution, since lists works in example news without problems, the problems is probably with the menu module or something...